projects/addon-editor/directives/tiptap-editor/tiptap-editor.directive.ts

```diff
@@ -26,7 +26,10 @@ import {TuiTiptapEditorService} from './tiptap-editor.service';
 export class TuiTiptapEditorDirective {
     @Input()
     set value(value: string) {
-        this.editor.setValue(value);
```

Comment:
Looks like this change should be in the service instead.

Reply:
fixed